source/features/mark-private-orgs.js

```diff
@@ -0,0 +1,44 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import {getCleanPathname} from '../libs/page-detect';
+import {getUsername} from '../libs/utils';
+import * as icons from '../libs/icons';
+import api from '../libs/api';
+
+export default async () => {
+	if (!getCleanPathname().startsWith(getUsername())) {
+		// Only for own user
+		return;
+	}
+	// List public orgs
+	const publicOrgs = [];
+	const orgDataList = await api(`users/${getUsername()}/orgs`, true);
```

Comment:
It would be better to get the information whether the org is private or not from the JSON payload, rather than doing an anonymous request. And authorized request has other benefits, like much higher rate limit, so we should prefer that whenever possible.

Reply:
Turns out this endpoint returns the public orgs, whether or not the user is signed in. No need for anon requests.